### PR TITLE
Add new stable charts

### PR DESCRIPTION
**Pull request by zunkree**
#### What this PR does / why we need it:
* Add chart for oidc-starter
* Add chart for oidc-static

### CODE REVIEW

File: stable/oidc-starter/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+    - host: {{ . | quote }}
+      http:
+        paths:
+	{{- range $ingressPaths }}
```
Comment (onorua) [inline stable/oidc-starter/templates/ingress.yaml]: tab vs spaces

File: stable/oidc-static/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+    - host: {{ . | quote }}
+      http:
+        paths:
+	{{- range $ingressPaths }}
```
Comment (onorua) [inline stable/oidc-static/templates/ingress.yaml]: tabs vs spaces

File: stable/oidc-static/values.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  init:
+    limits:
+      cpu: 16m
+      memory: 32Mi
+#    requests:
+#      cpu: 100m
+#      memory: 128Mi
+  nginx:
+    limits:
+      cpu: 16m
```
Comment (onorua) [inline stable/oidc-static/values.yaml]: are you sure that 16 milicors is enough?

File: stable/oidc-starter/values.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  hosts:
+  - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  limits:
+    cpu: 16m
```
Comment (onorua) [inline stable/oidc-starter/values.yaml]: are you sure that 16 millicors is enough? 
reference:
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu

File: stable/oidc-static/values.yaml (hunk 3 above)
Comment (zunkree) [reply stable/oidc-static/values.yaml]: looks so, and here it is a minimal set of resources to start app, for more we can override it in values when necessary

File: stable/oidc-starter/values.yaml (hunk 4 above)
Comment (zunkree) [reply stable/oidc-starter/values.yaml]: the same as above

**Pull request merged**